content/applications/sales/sales/send_quotations/create_quotations.rst

```diff
@@ -253,6 +253,25 @@ fields that can be configured.
   confirmation window appears, as well.
 - :guilabel:`Analytic Account`: Select an analytic account to apply to this customer/quotation.
 
+When :doc:`Debug mode is enabled <../../../general/developer_mode>`, additional fields appear in
+this section:
+
+- :guilabel:`Invoicing Journal`: The sales order invoices in the journal selected. When no selection
+  is made, the sales journal with the lowest sequence is used.
+- :guilabel:`Invoice Status`: This field **only** appears when the quotation has become a sales
```

Comment:
So, actually, just because you manually save it, that doesn't make it an official sales order. Yes, it does apply a 'SO###' at the top, but the status of the quotation remains in 'Quotation' -- it is only when the quotation is confirmed that it becomes an official sales order, and that other field appears.

Reply:
Ahh, ok- that makes sense! I was testing each click/method to see when I'd see the field, and/or when I could modify it. So thanks for explaining that- I thought it WAS a sales order once it had the number assigned!